ticdc/manage-ticdc.md

```diff
@@ -196,70 +343,6 @@ For nodes other than owner nodes, executing the above command will return the fo
 election: not leader
 ```
 
-### Stop replication task
-
-Use the following command to stop a replication task:
-
-{{< copyable "shell-regular" >}}
-
-```shell
-curl -X POST -d "admin-job=1&cf-id=28c43ffc-2316-4f4f-a70b-d1a7c59ba79f" http://127.0.0.1:8301/capture/owner/admin
-```
-
-```
-{
- "status": true,
- "message": ""
-}
-```
-
-In the above command:
-
-- `admin-job=1` means to stop the replication task. After the task is stopped, all `processor`s are stopped and exit but the configuration and status of the task are saved and can be recovered from `checkpoint-ts`.
-- `cf-id=xxx` is the ID of `changefeed` that needs operation.
-
-### Resume replication task
-
-Use the following command to resume a replication task:
-
-{{< copyable "shell-regular" >}}
-
-```shell
-curl -X POST -d "admin-job=2&cf-id=28c43ffc-2316-4f4f-a70b-d1a7c59ba79f" http://127.0.0.1:8301/capture/owner/admin
-```
-
-```
-{
- "status": true,
- "message": ""
-}
-```
-
-In the above command:
-
-- `admin-job=2` means to resume the replication task from `checkpoint-ts`.
-- `cf-id=xxx` is the ID of `changefeed` that needs operation.
-
-### Delete replication task
-
-Use the following command to delete a replication task:
-
-{{< copyable "shell-regular" >}}
-
-```shell
-curl -X POST -d "admin-job=3&cf-id=28c43ffc-2316-4f4f-a70b-d1a7c59ba79f" http://127.0.0.1:8301/capture/owner/admin
-```
-
-```
-{
- "status": true,
- "message": ""
-}
-```
-
-- `admin-job=3` means to delete the replication task. After the TiCDC server receives the request, all `processor`s are stopped and the configuration information of the task is cleared. The replication status is reserved. No service is available except for the query.
-- `cf-id=xxx` is the ID of `changefeed` that needs operation.
-
 ## Error handling
```

Comment:
I don't see this section in the Chinese doc: https://docs.pingcap.com/zh/tidb/dev/manage-ticdc.
Is it deleted?

Reply:
removed in 26d18ec595a4890ed699b0f1f838f7b5f78ceb9c